components/backstage/components/backstage_list.tsx

```diff
@@ -20,16 +25,29 @@ type Props = {
     helpText?: ReactNode;
     loading: boolean;
     searchPlaceholder?: string;
+    page?: number;
+    nextPage?: () => void;
+    previousPage?: () => void;
 };
 
 const BackstageList = ({searchPlaceholder = localizeMessage('backstage_list.search', 'Search'), ...remainingProps}: Props) => {
     const [filter, setFilter] = useState('');
 
     const updateFilter = (e: ChangeEvent<HTMLInputElement>) => setFilter(e.target.value);
 
+    const getPaginationProps = (total: number) => {
+        const page = remainingProps.page || 0;
```

Comment:
this function can be simplified by adding a range, activePage can be added to the state to manipulate the styling better, and using JS array methods.

Reply:
I like this idea @michelengelen. Could be a later enhancement for sure